File: python/시저 암호.py
def solution(s, n):
    lower_list = "abcdefghijklmnopqrstuvwxyz"
    upper_list = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    
    arr = []
    for elem in s:
        if elem == ' ':
            arr.append(' ')
        elif elem.islower() is True:
            append_idx = lower_list.find(elem) + n
            arr.append(lower_list[append_idx % 26])
        else:
            append_idx = upper_list.find(elem) + n
            arr.append(upper_list[append_idx % 26])
                
    return ''.join(arr)

File: js/시저 암호.js
function solution(s, n) {
    const lower = "abcdefghijklmnopqrstuvwxyz";
    const upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    var arr = [];
    
    for (let i=0; i<s.length; i++) {
        if (s[i] == ' ') {
            arr.push(' ');
        }
        else {
            // 대문자일 때
            if (s[i] === s[i].toUpperCase()) {
                let new_idx = [...upper].indexOf(s[i]) + n;
                arr.push([...upper][new_idx % 26]);
            } else {
                let new_idx = [...lower].indexOf(s[i]) + n;
                arr.push([...lower][new_idx % 26]);
            }
        }
    }
    return arr.join('');
}